serverlessworkflow/antora.yml

```diff
@@ -14,6 +14,7 @@ asciidoc:
     # upstream: remove this var
     #quarkus_cli_platform_redhat: -Pcom.redhat.quarkus.platform:quarkus-bom:2.7.6.Final-redhat-00006
     quarkus_cli_platform_redhat: ""
+    kogito_version: 2.0.0-SNAPSHOT
```

Comment:
don't we already have the kogito version set somewhere?

Reply:
I looked everywhere and didn't find it.

Reply:
found this one `kogito_version_redhat:`
isn't better to align the env name so it can be used for prod and community just by changing its value?
@ricardozanini  ^

Reply:
it's `kogito_version_redhat`. This variable is only valid downstream. Upstream we use the branch version: `page-version` (and others). See: https://docs.antora.org/antora/2.2/page/page-and-site-attributes/#page-attributes

Reply:
So, we don't need `kogito_version`. I'll remove it.
Thanks, guys.